packages/build-utils/src/types.ts

```diff
@@ -368,3 +369,49 @@ export interface ProjectSettings {
   directoryListing?: boolean;
   gitForkProtection?: boolean;
 }
+
+export interface FrameworkBuilder {
+  version: 2;
+  build: FrameworkBuild;
+  prepareCache?: PrepareCache;
+}
+
+export interface FunctionBuilder {
+  version: 3;
+  build: FunctionBuild;
+  prepareCache?: PrepareCache;
+  startDevServer?: StartDevServer;
+}
+
+type ImageFormat = 'image/avif' | 'image/webp';
+
+export interface Images {
+  domains: string[];
+  sizes: number[];
+  minimumCacheTTL?: number;
+  formats?: ImageFormat[];
+}
+
+export interface BuildResultV2 {
+  // TODO: needs routing-utils for `Route` type
+  //routes: Route[];
```

Comment:
```
yarn turbo run build --graph=graph.html 
```

![image](https://user-images.githubusercontent.com/229881/152443713-f17e802c-aece-4382-a39b-ec4b6a53905c.png)

I think we're going to get into circular dependencies, so it might be best to create a new package with just types? For the time being you can just use `routes: any[]` and we can fix it in a future PR.

Reply:
Updated to use `any[]` for now.